src/sentry/api/endpoints/internal/integration_proxy.py

```diff
@@ -169,6 +169,8 @@ def http_method_not_allowed(self, request):
         if not self._should_operate(request):
             raise Http404
 
+        metrics.incr("hybrid_cloud.integration_proxy.initialize")
```

Comment:
If this metric is low volume you might not get results due to sampling. You could set `sample_rate=1.0` to ensure that metrics are always collected here (and elsewhere) for low volume metrics.

Reply:
Makes sense, that is probably why I'm not seeing the DD metrics.